.changeset/early-goats-appear.md

```diff
@@ -0,0 +1,7 @@
+---
+"@frontity/wp-source": patch
+---
+
+- Fix wrong default totals when headers are missing. It was set to 0, but it has to be 1.
```

Comment:
Is this correct? Because in the code, I can see that the fallback is still `0`.

Reply:
Good catch, thanks!